internal/api/mail.go

```diff
@@ -649,9 +650,19 @@ func (a *API) sendEmail(r *http.Request, tx *storage.Connection, u *models.User,
 			SiteURL:         externalURL.String(),
 			TokenHash:       tokenHashWithPrefix,
 		}
-		if emailActionType == mail.EmailChangeVerification && config.Mailer.SecureEmailChangeEnabled && u.GetEmail() != "" {
-			emailData.TokenNew = otpNew
-			emailData.TokenHashNew = u.EmailChangeTokenCurrent
+		if emailActionType == mail.EmailChangeVerification {
+			if config.Mailer.SecureEmailChangeEnabled && u.GetEmail() != "" {
+				emailData.TokenNew = otpNew
+				emailData.TokenHashNew = u.EmailChangeTokenCurrent
+			} else if emailData.Token == "" && u.EmailChange != "" {
+
+				// BUG(cstockton): I am fixing this bug in a way that is
```

Comment:
Nit about this comment maybe reword to explain what needs to be considered in the future?

Reply:
I've attempted to clarify some here